src/components/Checkbox/Checkbox.tsx

```diff
@@ -33,21 +35,27 @@ let Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
             blockName = "",
             checkboxId,
             labelOptions,
-            isSelected = false,
+            checked,
+            defaultChecked,
```

Comment:
These two values are not being used because on lines 54, 55, and 58, you're using `props.checked` and `props.defaultChecked`.

Reply:
This should be fixed now. I removed the `props.` and just using `checked` and `defaultChecked` directly.